src/App.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import fetchGithubUser from './utils/githubFetch';
+import UserList from './Components/UserList';
+import './App.css';
+
+const mapToUsersArr = (str) => {
+  return str.replace(/\s+/g, '').split(',');
+};
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      userInput: '',
+      usersTofetch: [],
+      showUserList: false,
+      githubUsers: []
+    }
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleInputChange = this.handleInputChange.bind(this);
```

Comment:
I could use the babel `transform class properties` plugin which would allow me to define those methods in a way in which their `this` context is always bound to the `App` instance.

so instead of something like:

```javascript
handleSubmit(e) {
  // content goes here
}
```

I would have something like this instead:

```javascript
handleSubmit = (e) => {
  // this is bound to the App instance
}
```